### PR TITLE
Make dropdowns in top bar disappear after delay

### PR DESCRIPTION
**Pull request by 4i60r**

### CODE REVIEW

**Comment by ar4s** [comment]
:ok:

File: src/ralph/admin/templates/admin/includes/javascripts.html [hunk 1]
```diff
@@ -98,5 +98,26 @@
...
+            var dropdowns = parent.find("li.has-dropdown .dropdown");
+            dropdowns.each(function() {
+                var $drop = $(this);
+                var width = 'auto';
+                if(!$drop.is($thisdrop)) {
+                    width = '';
+                    $drop.hide();
+                    setTimeout(function() {
+                        $drop.show();
+                    }, 300);
```
Comment (mkurek) [inline src/ralph/admin/templates/admin/includes/javascripts.html]: isn't 300ms too much?

Comment (mkurek) [inline src/ralph/admin/templates/admin/includes/javascripts.html]: btw, could we do sth like this: if the cursor is further from dropdown then visibility time decreases? :)

File: src/ralph/static/src/scss/_overrides.scss [hunk 2]
```diff
@@ -85,3 +85,23 @@ select {
...
     overflow-x: auto;
 }
+
+.top-bar {
+  .top-bar-section {
+    li.has-dropdown {
+
+      &.hover, &.not-click {
+        > .dropdown {
+          transition: clip 0s 1s linear;
```
Comment (mkurek) [inline src/ralph/static/src/scss/_overrides.scss]: please change it to some smaller value, ex. 0.5s.

**Comment by mkurek** [comment]
make it mergable.

**Comment by szok** [comment]
🆗

**Comment by mkurek** [comment]
:ok:

**Pull request merged**